pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -199,26 +203,24 @@ let
       "zlib"
     ];
 
-    # Make sure Bazel knows about our configuration flags during fetching so that the
-    # relevant dependencies can be downloaded.
-    bazelFetchFlags = bazel-build.bazelBuildFlags;
-
-    bazelBuildFlags = [
+    bazelFlags = [
       "-c opt"
     ] ++ lib.optional (stdenv.targetPlatform.isx86_64 && stdenv.targetPlatform.isUnix) [
       "--config=avx_posix"
     ] ++ lib.optional cudaSupport [
       "--config=cuda"
     ] ++ lib.optional mklSupport [
       "--config=mkl_open_source_only"
+    ] ++ lib.optionals stdenv.cc.isClang [
+      "--cxxopt=-x" "--cxxopt=c++" "--host_cxxopt=-x" "--host_cxxopt=c++"
```

Comment:
OOC why were these flags chosen?

Reply:
This is due to https://github.com/NixOS/nixpkgs/issues/150655, it's a workaround that forces all files to be compiled as c++ files, I'll add a comment.